test/correctness/simd_op_check.cpp

```diff
@@ -1871,6 +1871,16 @@ struct Test {
         check("vmpa(v*.h,r*.b)", hvx_width/2, 2*i32(i16_1) + 3*i32(i16_2));
         check("v*.w += vmpa(v*.h,r*.b)", hvx_width/2, 2*i32(i16_1) + 3*i32(i16_2) + i32_1);
 
+        check("v*:*.h = vtmpy(v*:*.ub, r*.b)", hvx_width/1, 2*i16(in_u8(x + hvx_width - 1)) + 3*i16(in_u8(x + hvx_width)) + i16(in_u8(x + hvx_width + 1)));
```

Comment:
Is this adding hvx_width because x-1 reads out of bounds? We should just fix the simd_op_check driver so that the buffers have some padding before the min to enable checks like this, without having to shift them over by hvx_width.